docs/modules/operation/pages/performance-data-collection/collectors/SnmpCollector.adoc

```diff
@@ -47,7 +47,7 @@ Configure credentials to access SNMP agents in the Web UI (*Admin>Configure SNMP
 To configure which MIB objects to collect from devices, create a datacollection XML configuration file in `etc/datacollection/`.
 After creating a new datacollection config file, add it to `etc/datacollection-config.xml` for inclusion in polling.
 The `datacollection-config.xml` is reloaded automatically whenever the file is modified.
-The content of dataCollectionGroups in `etc/datacollection/*.xml` files are reloaded each time the Collectd process gathers metics from a node.
+The content of dataCollectionGroups in `etc/datacollection/*.xml` files are reloaded each time the Collectd process gathers metrics from a node.
```

Comment:
Will be fixed via https://github.com/OpenNMS/opennms/pull/3630 as this file is slated to move to a different folder.

Reply:
I've removed this change from the PR.